test/smoke.js

```diff
@@ -1,19 +1,25 @@
-const asciidoctor = require('asciidoctor.js')();
-require('../dist/main.js');
+const asciidoctor = require('asciidoctor.js')()
+const asciidoctorRevealjs = require('../dist/main.js')
+//require('../dist/main.js')
 
-const expect = require('expect.js');
+const expect = require('expect.js')
 
-const attributes = {'revealjsdir': 'node_modules/reveal.js@'};
-const options = {safe: 'safe', backend: 'revealjs', attributes: attributes, 'header_footer': true};
+// Register the reveal.js converter
+asciidoctorRevealjs.register()
+
+const attributes = {'revealjsdir': 'node_modules/reveal.js@'}
+const options = {safe: 'safe', backend: 'revealjs', attributes: attributes, 'header_footer': true}
 const content = `= Title Slide
 
 == Slide One
 
 * Foo
 * Bar
-* World`;
+* World`
+
+const result = asciidoctor.convert(content, options)
 
-const result = asciidoctor.convert(content, options);
+expect(asciidoctorRevealjs.getVersion()).to.be('1.1.4-dev')
```

Comment:
We should probably use a variable, otherwise we will have to update this value when we release a new version :disappointed:

Reply:
Sourced it out of `package.json` in 289e390. I think it should work in all cases.